great_expectations/execution_engine/sparkdf_execution_engine.py

```diff
@@ -591,14 +592,17 @@ def _combine_row_conditions(row_conditions: List[RowCondition]) -> RowCondition:
             condition=joined_condition, condition_type=RowConditionParserType.SPARK_SQL
         )
 
+    @public_api
     def get_compute_domain(
         self,
         domain_kwargs: dict,
         domain_type: Union[str, MetricDomainTypes],
         accessor_keys: Optional[Iterable[str]] = None,
     ) -> Tuple[DataFrame, dict, dict]:
-        """Uses a given batch dictionary and domain kwargs (which include a row condition and a condition parser)
-        to obtain and/or query a batch. Returns in the format of a Spark DataFrame.
+        """Uses a DataFrame and domain kwargs (which include a row condition and a condition parser) to obtain and/or query a Batch of data.
+
+        Returns in the format of a Spark DataFrame along with Domain arguments required for computing.  If the Domain
+        is a single column, this is added to 'accessor domain kwargs' and used for later access.
 
         Args:
             domain_kwargs (dict) - A dictionary consisting of the domain kwargs specifying which data to obtain
```

Comment:
Douple check the rendering, might need to use colons `:` instead. Also you can indent any lines following the first line of the arg for readability.